docs/user/alerting/defining-alerts.asciidoc

```diff
@@ -0,0 +1,80 @@
+[role="xpack"]
+[[defining-alerts]]
+== Defining alerts
+
+{kib} alerts can be created in a variety of apps including <<xpack-apm,*APM*>>, <<xpack-infra,*Metrics*>>, <<xpack-siem,*SIEM*>>, <<xpack-uptime,*Uptime*>> and from <<management,*Management*>> UI. While alerting details may differ from app to app, they share a common interface for defining and configuring alerts that this section describes in more detail. 
```

Comment:
This first sentence appears in a lot of places. I don't think it is needed here.